hpi-schul-cloud/nuxt-client
# BC-4965 - Remove consent reference from student visibility toggle info text

# Short Description

QA note:
Change tested and checked. Changes only for BRB. No issues found. 
See ticket: https://ticketsystem.dbildungscloud.de/browse/BC-4965 





## Links to Ticket and related Pull-Requests
https://ticketsystem.dbildungscloud.de/browse/BC-4965
https://github.com/hpi-schul-cloud/schulcloud-client/pull/3297


## Changes



## Data-security



## Deployment



## New Repos, NPM packages or vendor scripts



## Screenshots of UI changes



## Checklist before merging


> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed
